docs/api.rst

```diff
@@ -1,6154 +1,10 @@
 .. SPDX-License-Identifier: MIT
 
-.. currentmodule:: disnake
+:orphan:
 
-API Reference
-===============
+Legacy Page
+===========
```

Comment:
```suggestion
Redirecting ...
===============
```
Seeing `Legacy Page` for a split second before being redirected might be confusing, thoughts on using `Redirecting ...` instead?
(same thing in `ext/commands/api.rst`)

Reply:
https://github.com/DisnakeDev/disnake/pull/722/commits/8f41342ae696631cfd9a6b6a1c02e2b85f41a0e3